test-cases/terraform/aws/encryption/at-rest/workspace_user_volume_not_encrypted_at_rest/results_summary.json

```diff
@@ -4,5 +4,5 @@
   "kics": "no",
   "snyk": "no",
   "terrascan": "no",
-  "tfsec": "no"
+  "tfsec": "yes"
```

Comment:
This isn't technically a "yes" because this test case focuses on the user volume, whereas the tfsec output only has the root volume.

Reply:
yup, thats a typo on my part in the check, which I'll fix now

Reply:
corrected messages